### PR TITLE
Updates related to removal of UWP, Xamarin, NET6-Win support in 4.61.0

### LINKED ISSUE #395: Update the guidance on using WAM and embedded browser for net6-windows apps
After MSAL 4.61.0 is released which removes net6.0-windows target, update the WAM and embedded browser usage guidance in the docs:

- https://learn.microsoft.com/en-us/entra/msal/dotnet/acquiring-tokens/using-web-browsers#browser-availability
- https://learn.microsoft.com/en-us/entra/msal/dotnet/acquiring-tokens/desktop-mobile/wam#package-availability

I would probably still mention net6-windows category, just add clarification that behavior changed in 4.61.0.

### PR DESCRIPTION
**Pull request by pmaytak**
Fixes #395

### CODE REVIEW

File: msal-dotnet-articles/acquiring-tokens/using-web-browsers.md [hunk 1]
```diff
@@ -41,25 +41,27 @@ Generally, it's recommended that you use the platform default, and this is typic
...
+| Xamarin.Android††††             | ✅ Yes                   | ✅ Yes                | System                |
+| Xamarin.iOS††††                 | ✅ Yes                   | ✅ Yes                | System                |
+
+**†** System browser requires `http://localhost` redirect URI.
 
 **††** Target `net6.0-windows` or above to use the embedded browser.
 
 **†††** Reference [Microsoft.Identity.Client.Desktop](https://www.nuget.org/packages/Microsoft.Identity.Client.Desktop) and call <xref:Microsoft.Identity.Client.Desktop.DesktopExtensions.WithWindowsDesktopFeatures%2A> to use the embedded browser.
 
+**††††**MSAL.NET versions 4.61.0 and above do not provide support for UWP, Xamarin Android, and Xamarin iOS.
```
Comment (pmaytak) [inline msal-dotnet-articles/acquiring-tokens/using-web-browsers.md:63]:
```suggestion
**††††** MSAL.NET versions 4.61.0 and above do not provide support for UWP, Xamarin Android, and Xamarin iOS.
```

File: msal-dotnet-articles/acquiring-tokens/desktop-mobile/wam.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ WAM support is split across two packages:
 >[!NOTE]
 >For migration purposes, and if you have a .NET 6, .NET Core, or a .NET Standard application that needs to use _both_ WAM and the [embedded browser](/azure/active-directory/develop/msal-net-web-browsers#embedded-vs-system-web-ui), you will also need to use the [Microsoft.Identity.Client.Desktop](https://www.nuget.org/packages/Microsoft.Identity.Client.Desktop/) package. Once added, developers can use [`WithWindowsDesktopFeatures`](xref:Microsoft.Identity.Client.Desktop.DesktopExtensions.WithWindowsDesktopFeatures*) when setting up their public client application.
 >
->If your application targets UWP or `net-windows` (version-dependent Target Framework Moniker for Windows), WAM is included in the MSAL.NET package.
+>If your application targets UWP or `net-windows` (version-dependent Target Framework Moniker for Windows), WAM is included in the MSAL.NET package (for versions 4.60.3 or below).
```
Comment (pmaytak) [inline msal-dotnet-articles/acquiring-tokens/desktop-mobile/wam.md:36]:
```suggestion
>If your application targets UWP or `net-windows` (version-dependent Target Framework Moniker for Windows), WAM is included in the MSAL.NET package (applicable to versions 4.60.3 and below).
```

**Review by localden — APPROVED** [review]

**Review by gladjohn — CHANGES_REQUESTED** [review]
this needs an update 

https://learn.microsoft.com/en-us/entra/msal/dotnet/acquiring-tokens/desktop-mobile/wam#enabling-wam

remove this `If your application targets UWP or net-windows (version-dependent Target Framework Moniker for Windows), WAM is included in the MSAL.NET package.`

**Comment by pmaytak** [comment]
> this needs an update
> 
> https://learn.microsoft.com/en-us/entra/msal/dotnet/acquiring-tokens/desktop-mobile/wam#enabling-wam
> 
> remove this `If your application targets UWP or net-windows (version-dependent Target Framework Moniker for Windows), WAM is included in the MSAL.NET package.`

@gladjohn Yes, I specifically left it to not confuse folks on older versions. I added a comment that it's applicable to previous versions.

>If your application targets UWP or `net-windows` (version-dependent Target Framework Moniker for Windows), WAM is included in the MSAL.NET package **(applicable to versions 4.60.3 and below).**

**Pull request merged**